static/js/index.js

```diff
@@ -89,14 +112,16 @@ function sendUpdatePrivacyRequest(share_photo, share_schedule) {
       if (!result.error) {
         renderToast("Settings updated!");
       } else {
-        console.log(result.error);
+        console.log(result.error); // logs error if update privace fails
         renderToast(result.error);
       }
     }
   }
   xhr.open("POST", "privacy", true);
   xhr.send(data);
 }
+
+// idk what this code does -- Probably related the 20-21 school year senior prank (depreciated code point towards this)
```

Comment:
This is for keyboard handling for left/right and escape. Nothing to do with any pranks...